manager_rethink.go

```diff
@@ -70,6 +71,7 @@ type RethinkManager struct {
 	Policies map[string]Policy
 }
 
+// ColdStart loads all policies from rethinkdb without warming up the cache
```

Comment:
Ah I see, ColdStart is normally used when the policy map of the RethinkDB Manager is empty (usually after restart) :)